alpha/declcfg/load.go

```diff
@@ -213,10 +261,10 @@ func formatUnmarshallErrorString(data []byte, errmsg string, offset int64) strin
 		var pOffset, origOffset int64
 		origOffset = 0
 		for origOffset = 0; origOffset < offset; {
-			pOffset++
 			if pString[pOffset] != '\n' && pString[pOffset] != ' ' {
 				origOffset++
 			}
+			pOffset++
```

Comment:
Is there a reason this was moved? Could it have been intentional that the `pOffset` increment occurs before it's use?

Reply:
Yeah I moved it because I was getting off-by-one panics in the tests I implemented, which included an error at the last byte of the JSON string, and as this tried to map to the pretty-printed offset, it incremented `pOffset` beyond the length of `pString`.

Maybe @grokspawn can confirm for sure if there's something we should be aware of. But I tested before an after of this change to make sure the resulting error looked the same. All the other tests also still passed, so I'm thinking I was just the unlucky person to run into this untested edge case?